src/validators/rdf.js

```diff
@@ -40,7 +40,7 @@ export default class RDFScanner {
         errorHandler: (err) => {
           reject(new RDFParseError(`RDF Parse Error: ${err}`));
         },
-      }).parseFromString(this.contents);
+      }).parseFromString(this.contents, 'text/xml');
```

Comment:
Darn, nope, that crept in. It’s not even needed, was just being more explicit there. I’ll take it out :-)